docs/plugins.md

```diff
@@ -202,7 +202,7 @@ pipx inject poetry poetry-plugin
 If you want to uninstall a plugin, you can run:
 
 ```shell
-pipx runpip poetry uninstall poetry-plugin
+pipx uninject poetry poetry-plugin
```

Comment:
Please leave both options with a note about `pipx` versions.

Reply:
Okay, will do. Just as an FYI this feature was added about a year ago.